frontend/src/metabase/querying/notebook/components/DataStep/DataStep.unit.spec.tsx

```diff
@@ -74,6 +75,19 @@ const setup = ({
   setupSearchEndpoints([]);
   setupRecentViewsAndSelectionsEndpoints([], ["selections"]);
 
+  // In embedding SDK we call a different endpoint because we use a different data picker (metabase#52889)
+  if (isEmbeddingSdk) {
+    setupSearchEndpoints(
+      [
+        createMockModelResult({
+          id: 0,
+          name: "Products",
+        }),
+      ],
+      { overwriteRoutes: true },
```

Comment:
Is this because we did `setupSearchEndpoints([]);` above so it needs to override our mock search endpoints?

Reply:
yes

Reply:
Would it make sense to call `setupSearchEndpoints([]);` in a `else` statement here instead to make explicit?

Reply:
I think I like your idea, the explicitness is worth it.

Fixed in 01ec5f1ecc1c72264089a364b63c914b59a8d7dd